### PR TITLE
FS-2542 Updating contact us page to match gov uk design system

### PR DESCRIPTION
**Pull request by srh-sloan**
### Change description
Updated the partial page used to render contact details to match the gov-uk design system https://design-system.service.gov.uk/patterns/contact-a-department-or-service-team/

Updated translations as the headings for email, telephone etc no longer include a `:`

Translations pending for 'except public holidays' but this will be picked up in [FS-2532](https://digital.dclg.gov.uk/jira/browse/FS-2532) when the rest of the contact us page is fully translated.

- [n/a] Unit tests and other appropriate tests added or updated
- [n/a] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
Run locally using the docker runner and navigate to http://localhost:3008/contact_us to see the changes

### Screenshots of UI changes (if applicable)
Screenshot 2023-04-20 at 13 29 36

### CODE REVIEW

File: app/translations/cy/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -2474,3 +2453,27 @@ msgstr "o'r meini prawf asesu"
...
+#~ "wnaethom eich gwahodd, "
+
+#~ msgid "express your interest in the fund"
+#~ msgstr "mynegwch eich diddordeb yn y gronfa yn gyntaf"
+
+#~ msgid "Opening times:"
+#~ msgstr "Amserau agor:"
+
+#~ msgid "Closed:"
+#~ msgstr ""
```
Comment (Adam-W1) [inline app/translations/cy/LC_MESSAGES/messages.po:2479]: is this section meant to be commented out?

Comment (srh-sloan) [reply app/translations/cy/LC_MESSAGES/messages.po:2479]: Yes - these are auto generated and it means last time we updated translations these were needed, but now they are not. Next time we run update if they still aren't there they will be removed.

Comment (Adam-W1) [reply app/translations/cy/LC_MESSAGES/messages.po:2479]: ah okay - good to know 👍

**Review by Adam-W1 — APPROVED** [review]
looks good 🐉

**Review by Adam-W1 — APPROVED** [review]

**Pull request merged**